src/AddTodoForm.js

```diff
@@ -24,14 +25,9 @@ function AddTodoForm( { onAddTodo }) {
     }
     return (
         <form id="todoForm" onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle">Title</label>
-            <input 
-            name = "title" 
-            type = "text" 
-            id="todoTitle" 
-            value={todoTitle} 
-            onChange={handleTitleChange}>
-            </input>
+            <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange}>
+                Title
+            </InputWithLabel>
```

Comment:
looking good here! 👍 

You're only missing the `id` and `isFocused` prop here. The `id` prop will allow you two or more instances of the InputWithLabel to be created. Also, you can still pass in an isFocused prop so the component knows when it should autoFocus.